### DISCUSSION TITLE
Inverse color palette

### DISCUSSION THREAD

**Discussion by NielsMeijer234838** [mwaskom/seaborn#3621] [Q&A]
I have an jupyter notebook where I am trying to generate and export some visualisations. However, the color palette is inversed compared to the documentations. I only ran the imports, loading of the data and the code below to exclude any overseen code. 

If it is a bug with my IDE how would I go about preventing it without changing to many settings?

```
sample = df.sample(frac=0.1, random_state=199)
plt.figure(figsize=(8, 8))
bright_cmap = sns.color_palette("Spectral", as_cmap=True)
heatmap = sns.heatmap(sample[['Offensive duels per 90', 'Deep completions per 90', 'Shots per 90', 'Shots on target, %', 'Minutes played', 'Successful attacking actions per 90', 'xG per 90', 'Penalty conversion, %', ]].corr(), annot=True, cmap=bright_cmap, fmt=".2f")
plt.show()
```

![Image](https://github.com/mwaskom/seaborn/assets/144094547/9171bb9f-c59c-422b-b847-0bf989a1b002)

**Comment by mwaskom** [comment]
Looks like your IDE is in dark mode, maybe it just inverts colors when it does that?

**Reply by NielsMeijer234838** [reply]
That does revert it to the correct colors.

**Comment by NielsMeijer234838** [comment]
Jetbrains IDE implemented themes by putting an inverse color filter over the application, this works fine unless the output is directly being shown in the IDE as is the case with python notebooks. There already is a bug report on this. 

While they fix their IDE is there a seaborn feature that I am unaware off that can still get me the correct outputs without having to switch to light mode?
